components/src/md3/Hero/Hero.tsx

```diff
@@ -119,6 +119,7 @@ export const Hero: React.FC<HeroProps> = (props) => {
                         backgroundColor: iconBackgroundColor || theme.colors.surface,
                         height: typeof iconSize === 'number' ? normalizeIconSize() : iconSize ?? 36 * fontScale,
                         width: typeof iconSize === 'number' ? normalizeIconSize() : iconSize ?? 36 * fontScale,
+                        borderRadius:25,
```

Comment:
use `borderRadius : 50` to match react component library. https://github.com/etn-ccis/blui-react-component-library/blob/dev/components/src/core/Hero/Hero.tsx#L76

Reply:
Can we move this style [here](https://github.com/etn-ccis/blui-react-native-component-library/blob/release/md3/components/src/md3/Hero/Hero.tsx#L26C9-L26C20)?